index.js

```diff
@@ -19,7 +19,9 @@ const optsSchema = {
       ],
       default: {}
     },
-    env: { type: 'boolean', default: true }
+    env: { type: 'boolean', default: true },
+    dotenv: { type: 'boolean', default: false },
+    dotenvConfig: { type: 'object' }
```

Comment:
I think `dotenvConfig` is enough

Reply:
IMHO the only presence of the `dotenv` property is enough in order to enable this feature. Or I missing something?

Reply:
Ahh.. yeah I think you're right, it can totally work with just a single `dotenv`. I was distracted by the thought that "enabling of the feature" should be separate from "passing in the config options" but it can actually be combined like you said - good call.

I'll update PR to act like this:

This will enable the feature with default options:
```js
{
  ...
  dotenv: true
}
```

This will enable the feature _and_ pass in the options
```js
{
  ...
  dotenv: { path }
}
```

💥